### ISSUE TITLE
Auto: Investigate unclassified – Update process_pdfs.py

### ISSUE THREAD

**Issue by nehaa0305** [nehaa0305/Adobe_India_Hackathon_Team_MnN_Challenge_1a#1]
### Summary
ML generation failed

**Impact**: severity `Medium` | modules: n/a

### Affected modules
- (not inferred)

### Files changed
- `process_pdfs.py` (+1/-0)

### Diff excerpt: `process_pdfs.py`
```diff
@@ -14,6 +14,7 @@
 import re
 import cv2
 from PIL import Image
+import keras
 import layoutparser as lp
 import torch
 import pandas as pd
```

### Suggested labels
`unclassified`

### Suggested severity
`Medium`

### Reproduction (template)
1. Describe the environment/inputs
2. Steps to reproduce
3. Expected vs. actual behavior

### Notes
- Commit: `224a0b31` (https://github.com/nehaa0305/Adobe_India_Hackathon_Team_MnN_Challenge_1a/commit/224a0b31867ec3a8868d18975ff14f5e32362e88)
- Author: nehaa0305

### Proposed next steps
- [ ] Add/extend tests for the modified paths
- [ ] Validate edge cases and error handling
- [ ] Consider performance/memory impact if applicable
- [ ] Update documentation if behavior changed

---
### Diff excerpt: `process_pdfs.py`
```diff
@@ -14,6 +14,7 @@
 import re
 import cv2
 from PIL import Image
+import keras
 import layoutparser as lp
 import torch
 import pandas as pd
```
